seeds/categories.json

```
@@ -0,0 +1,20 @@
+[
+    {
+      "category": {
+        "slug": "facial-products",
+        "name": "Facial Products"
+      }  
+    },
+    {
+      "category": {
+        "slug": "hair-products",
+        "name": "Hair Products"
+      }    
+    },
+    {
+      "category": {
+        "slug": "body-products",
+        "name": "Body Products"
+      }    
+    }
```

The seeder is expecting the json file to contain a list `create` `/categories` request bodies.
Should be something like if I'm not mistaken.
```suggestion
    {
        "slug": "facial-products",
        "name": "Facial Products"
    },
    {
        "slug": "hair-products",
        "name": "Hair Products"
    },
    {
        "slug": "body-products",
        "name": "Body Products"
    }
```

---

Yeah the "category" key means the request looks like this:

```
$payload = [
  'category' => [
    'slug' => '...
    'name' => ''...
  ]
];
```

In products we want that, because the product API has a section of request attributes under the "product" key, but for categories we don't need that 👍